README.md

```diff
@@ -288,4 +305,12 @@ to be installed and enabled on the RabbitMQ server.
 
 Library creates a delayed message exchange to route messages back to the source queue with a set delay.
 
-## Enjoy!
+## [Known Issues](#table-of-contents)
+
+### [Several handlers listening to the same event](#table-of-contents)
+
+**Problem:** When several handlers listen to the same event, each handler receives the same event several times (equal to number of listeners)
+
+**Workaround**: You can create one "proxy" pub-sub event listener which will publish a local event with the same content. Then you can create as many event listeners to this local event as you need. Main disadvantage of this approach is that the pub-sub event is acknowledged in this "proxy" event listener. Therefore, if something goes wrong in the actual event listeners, the library won't handle the error.
```

Comment:
@Leichtwind 
there is a 2nd workaround:
instead of publishing a local event, you can execute a dedicated command.
in case if command fails the proxy event also fails

Reply:
But then all the commands are going to be retried or you need to handle it somehow

Reply:
this is just another workaround that you can add, so developer can choose which approach to use